src/common-ssh/asciicast.c

```diff
@@ -98,18 +98,8 @@ asciicast_recording* asciicast_recording_create(char* path, char* name, int heig
         return NULL;
     }
 
-    /* Create tmp file that indicates a recording in progress */
-    char* ext_tmp = ".cast.tmp";
-    char tmp_name[strlen(name) + strlen(ext_tmp) + 1];
-
-    if (snprintf(tmp_name, sizeof(tmp_name), "%s%s", name, ext_tmp) != strlen(tmp_name)) {
-        guac_client_log(client, GUAC_LOG_ERROR,
-                "Error creating tmp cast file name for: %s", name);
-        return NULL;
-    }
-
     /* Attempt to open recording file */
-    int fd = guac_recording_open(path, tmp_name, filename, sizeof(filename));
+    int fd = guac_recording_open(path, name, filename, sizeof(filename));
     if (fd == -1) {
         guac_client_log(client, GUAC_LOG_ERROR,
                 "Creation of recording failed: %s", strerror(errno));
```

Comment:
Shouldn't the message change?

Reply:
to what?

Reply:
"Failed to open recording file '%s': %s, name, strerror(errno)"